editor/app/components/utils.js

```diff
@@ -0,0 +1,15 @@
+export const buildStyles = (styles) => {
```

Comment:
@jandrade you had two questions around CSS.

1. Using [CSS modules](https://github.com/css-modules/css-modules).
Im definitely not opposed to it!  But I wonder if starting with just CSS object will suffice for now.  But I'd be happy if we explored this tho.

2. Does pakit support importing CSS files?
The answer is yes, you can import CSS files into your JS out of the box.  There should be no need to tweak anything.  Although it doesn't do anything in particular to scope, it just injects stuff into the dom for you.  I'd have to take a look to understand what functionality is missing.
You can see an example here https://github.com/MiguelCastillo/pakit/blob/5281bf98dc9dc6686e84fd5b74919aa13506cd99/examples/webapp/src/hello.jsx

Reply:
Oh I think that # 2 is more than enough! I think I tried that approach but probably did something wrong. I tried it again and it works perfectly fine. In that case 2 covers our use case more than enough :). You'll see the implementation that I'm doing for the `Slider` component in a next PR.

Re: Using CSS modules, starting with the CSS object is more than enough.